pkg/kubelet/apis/kubeletconfig/v1beta1/types.go

```diff
@@ -107,6 +107,13 @@ type KubeletConfiguration struct {
 	// Default: ""
 	// +optional
 	TLSMinVersion string `json:"tlsMinVersion,omitempty"`
+	// serverTLSBootstrap enables server certificate bootstrap. Instead of self
+	// signing a serving certificates, the kubelet will request a certificate
+	// from the certificates.k8s.io API. This requires the an approver to approve
+	// the certificate signing requests. The RotateKubeletServerCertificate
+	// feature must be enabled.
+	// Default: false
```

Comment:
Something that initially confused me: this is for `RotateKubeletServerCertificate`, not `RotateKubeletClientCertificate`. The former is still alpha, and the field is correctly feature-gated, so this behavior change (no longer try to bootstrap server tls by default) should be ok, even though we're on `v1beta1.KubeletConfiguration` now.